rust/main/lander/src/dispatcher/metrics.rs

```diff
@@ -197,6 +224,16 @@ impl DispatcherMetrics {
             .inc();
     }
 
+    pub fn get_finalized_nonce(&self, destination: &str) -> IntGauge {
+        self.finalized_nonce
+            .with_label_values(&[destination])
+            .clone()
+    }
+
+    pub fn get_upper_nonce(&self, destination: &str) -> IntGauge {
+        self.upper_nonce.with_label_values(&[destination]).clone()
```

Comment:
we probably need the signer in here as well

Reply:
Added